src/Stripe.net/Entities/Checkout/Sessions/SessionDisplayItem.cs

```diff
@@ -0,0 +1,49 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class SessionDisplayItem : StripeEntity
```

Comment:
The spec defines four different types `payment_pages_checkout_session_checkout_session_resource_custom_display_item`, `payment_pages_checkout_session_checkout_session_resource_plan_display_item`, `payment_pages_checkout_session_checkout_session_resource_sku_display_item` and `payment_pages_checkout_session_checkout_session_resource_subscription_data_display_item` (whew). In theory we should define four different classes that implement a common interface `ISessionDisplayItem`, rather than a single class that implements all possible fields.

Reply:
Hmmmm that's new? This is not the way we did `PaymentMethod`, why didn't you mention this at the time? Is this something we should have done in that case too?

Reply:
You'll have to be more specific, which polymorphic field are you referring to exactly?

Reply:
After discussing it internally, we do want this to be one resource after all and the current implementation should be correct.

Reply:
👍